test/rectest/getStepOptions_options.go

```diff
@@ -1,17 +1,3 @@
-// Copyright 2020 IBM Corp.
```

Comment:
Copyright should stay. Let's revert all the files where it's accidently removed.